### PR TITLE
Remove analytics server

### LINKED ISSUE #579: Remove the analytics service
## Problem

Currently, the analytics service isn't enabled in the production frontend, and we feel that most of the things we are collecting can be gleaned from the API request logs. As discussed previously we have concerns about the Analytics server modifying the API database (#484). Additionally, we'd like any telemetry/analytics of this kind to be opt-in and very transparent to our users, which will require time to build out UI elements for.

## Description

We should remove the analytics service entirely from this repo.

## Additional context

Alongside efforts to deprecate the ingestion server and replace it with a DAG, this would help the API repository to cease being monorepo-esque, which should ease maintenance. 

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes #579 by @zackkrida

## Description

This PR removes the analytics service package, removes all the mentions of analytics server from the docs, and removes analytics setup from `docker` settings/scripts.

## Testing Instructions

Run the API. It should start and work without any errors.

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [ ] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by sarayourfriend — APPROVED** [review]
Yesssss :tada:

**Review by sarayourfriend — COMMENTED** [review]
I guess we would also need changes in the nginx configuration in production to remove the analytics mapping?

**Comment by obulat** [comment]
> I guess we would also need changes in the nginx configuration in production to remove the analytics mapping?

Yes, but I have no idea what to do about it.

**Review by dhruvkb — APPROVED** [review]
Looks good to me! Thanks for also removing the Just recipes and doc mentions.

File: api/docs/guides/quickstart.md [hunk 1]
```diff
@@ -83,7 +83,6 @@ The command `just up` spawns the following services:
 - [NGINX](http://nginx.org)
 - **web** (`api/`)
 - **ingestion_server** and **indexer_worker** (`ingestion_server/`)
-- **analytics** (`analytics/`)
 
 The last three are subprojects of this monorepo.
```
Comment (stacimc) [inline api/docs/guides/quickstart.md:87]:
```suggestion
The last two are subprojects of this monorepo.
```

**Review by stacimc — COMMENTED** [review]
Love seeing all that red 🎉 Just one tiny docs comment, but this looks great!

**Review by stacimc — APPROVED** [review]

**Pull request merged**